helix-core/src/main/java/org/apache/helix/model/Message.java

```diff
@@ -117,6 +120,11 @@
   // Currently, the field is only used for invalidating messages in controller's message cache.
   private boolean _expired = false;
 
+  // The expect period of time (in ms) that a message should be completed, default 1 day
+  public static final long MESSAGE_EXPECT_COMPLETION_PERIOD = HelixUtil
```

Comment:
Do we want to have this param for our internal message report or including for some user defined messages? It is hard to define the time from user perspective.

Reply:
We have message.setExpectedCompleteTime(), if user would like to set their customer time for different message, they can do that, right?